source/switching_to_live/index.html.md.erb

```diff
@@ -63,7 +63,17 @@ If you use Worldpay and you want to accept American Express cards, you must cont
 
 You must send your American Express merchant ID to Government Banking or Worldpay if you’re a non-Crown merchant.
 
-## 4. Check your live account works
+## 4. Check your settings match your test account
+
+You need to make the following changes to your live account if you made them to your test account:
```

Comment:
Technically they might have made some changes to their test account that they actually don't want in the live account. That's probably clear and fine to leave if we think it is, but it might be good to clarify that we mean - if you made these changes in test and you want them in live, you'll need to make the changes in live as well.  ?

Reply:
@miriamalice As discussed we'll leave as is for now on the basis that most teams will want their live accounts to match their test accounts. 

This has been 2i'd now - could you green-tick and then I'll merge it in :)